Makefile

```diff
@@ -50,6 +50,13 @@ build-for-library-evolution:
 		-Xswiftc -emit-module-interface \
 		-Xswiftc -enable-library-evolution
 
+	swift build \
```

Comment:
Noting that this really only covers Library Evolution on `macOS`, but is an improvement on what's currently setup for CI.